setup.cfg

```diff
@@ -0,0 +1,58 @@
+[metadata]
+name = paramnb
+version = attr: param.version.get_setupcfg_version
+description = Generate ipywidgets from Parameterized objects in the notebook
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = BSD 3-Clause License
+license_file = LICENSE.txt
+classifiers =
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Development Status :: 4 - Beta
+author = PyViz
+author_email = holoviews@gmail.com
+maintainer = PyViz
+maintainer_email = holoviews@gmail.com
+url = https://github.com/ioam/paramnb
+project_urls =
+    Bug Tracker = https://github.com/ioam/paramnb/issues
+    Documentation = https://github.com/ioam/paramnb
```

Comment:
Not paramnb.pyviz.org?

Reply:
Bit confused. I don't know what paramnb.pyviz.org is or why we'd want anyone to look at it.

Reply:
Oh wait, now I see your comment on the parambokeh PR.

paramnb doesn't have a website, and can't currently have one generated by nbsite (https://github.com/pyviz/nbsite/issues/26).

Reply:
My suggestion would be to build the paramnb website manually one more time and embedding the widgets that way. I consider paramnb to be in maintenance mode after this release so I don't expect it will need updating very often. That also means there's no need for fqdn since deployment will be manual.

Reply:
Hmm.  I don't actually know what's at paramnb.pyviz.org, either.  It seems to be an incomplete build where not much worked, not even just lacking widgets.  It looked ok from the top level, and I didn't dive any deeper.